src/Roslyn.Diagnostics.Analyzers/UnitTests/PreferNullLiteralTests.cs

```diff
@@ -0,0 +1,264 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
```

Comment:
Can you please add at least one sanity test for VB?

Reply:
➡️ Covered by https://github.com/dotnet/roslyn-analyzers/pull/3372#discussion_r393687493